SearchList.js

```diff
@@ -634,6 +608,33 @@ export default class SearchList extends Component {
   }
 }
 
+SearchList.propTypes = {
```

Comment:
Your move all static properties out of the class, do your have any problem when using static properties?

Reply:
Nope, its just cleaner this way, at least for me, I can put them back in class if u want but its the same thing..